tests/publiccloud/azure_cli.pm

```diff
@@ -54,10 +54,14 @@ sub run {
     # VM creation
     my $vm_create = "az vm create --resource-group $resource_group --name $machine_name --public-ip-sku Standard --tags '$tags'";
     $vm_create .= " --image $image_name --size Standard_B1ms --admin-username azureuser --ssh-key-values ~/.ssh/id_rsa.pub";
-    my $output = script_output($vm_create, timeout => 600, proceed_on_failure => 1);
-    if ($output =~ /ValidationError.*object has no attribute/) {
-        record_soft_failure('bsc#1191482 - Failed to start/stop vms with azure cli');
-        return;
+    if (script_run($vm_create) != 0, timeout => 600, die_on_timeout => 1) {
```

Comment:
Why not simply use `assert_script_run`? :thinking: Would save us this if block entirely with the same effect.

Reply:
agree with Felix